hoot-services/src/main/java/hoot/services/controllers/export/ExportCommand.java

```diff
@@ -110,23 +120,46 @@ else if (translation.equalsIgnoreCase("translations/TDSv40.js")) {
 
     List<String> getCommonExportHootOptions() {
         List<String> options = new LinkedList<>();
-        options.add("convert.ops=hoot::DecomposeBuildingRelationsVisitor");
-        options.add("hootapi.db.writer.overwrite.map=true");
+
+        //Sets the user reading this data
         options.add("api.db.email=" + params.getUserEmail());
-        options.add("job.id=" + jobId);
 
-        //# Add the option to have status tags as text with "Input1" instead of "1" or "Unknown1"
-        if (params.getTextStatus()) {
-            options.add("writer.text.status=true");
+        //Convert ops, ORDER IS IMPORTANT
+        //we want to translate last
+        List<String> convertOps = new ArrayList<>();
+
+        //Remove review relations
+        if (!params.getIncludeHootTags()) {
+            convertOps.add("hoot::RemoveElementsVisitor");
+            options.add("remove.elements.visitor.element.criteria=hoot::ReviewRelationCriterion");
+            options.add("remove.elements.visitor.recursive=false");
+            options.add("writer.include.circular.error.tags=false");  //Not currently working for shp writer
+
+            convertOps.add("hoot::RemoveTagsVisitor");
+            options.add("remove.tags.visitor.keys=hoot:status;hoot:building:match;error:circular");
+
+            //TODO: Do we need to remove matched review features?
```

Comment:
do we?

Reply:
I guess not.  Though we do intend to remove matched review features before using a merged dataset as an input source for a subsequent conflation to avoid potential duplicates.